build-system/linux-pr-validation.yaml

```diff
@@ -0,0 +1,26 @@
+# Pull request validation for Linux against the `dev` and `master` branches
```

Comment:
Just add these as steps to the original `pr-validation.yaml` file rather than creating a new separate pipeline. Easier to manage that way (can change pipeline construction without having to add / remove new ones through Azure DevOps UI)